src/Tests/README.md

```diff
@@ -45,6 +45,18 @@ src/
       ...
 ```
 
+## Misc in tests
+
+Some names are used to filter specific tests.
+
+```ts
+suite('@Use-onecc', function() {
+  // ...
+});
+```
+
+Like `@Use-onecc` is expressing that these tests use `onecc` commands.
```

Comment:
I wonder the purpose of this naming. 
It would be great if we add the reason why we need to use this naming. What happens if we don't use this name by mistake?

Reply:
Nothing happens. However, if we use this name by mistake when we don't use `onecc`, the test suite would be not tested.

Reply:
https://github.com/Samsung/ONE-vscode/pull/424/files#r838176015